test/test_suite_test.go

```diff
@@ -144,19 +144,33 @@ var _ = BeforeSuite(func() {
 })
 
 var _ = AfterSuite(func() {
+
+	var destroy bool = true
 	if !helpers.IsRunningOnJenkins() {
 		log.Infof("AfterSuite: is dev env, keep templates as it is")
-		return
+		destroy = false
```

Comment:
I refer the early return than the addition of this new flag. What motivates the need for this new logic?

Reply:
Mainly on AfterSuite I want to run `GatherLogs` so I need to create this flag.